bftengine/src/bftengine/SigManager.cpp

```diff
@@ -199,7 +199,7 @@ bool SigManager::verifySig(
     if (auto pos = verifiers_.find(pid); pos != verifiers_.end()) {
       result = pos->second->verify(str_data, str_sig);
     } else {
-      LOG_ERROR(GL, "Unrecognized pid " << pid);
+      LOG_WARN(GL, "Unrecognized pid " << pid);
```

Comment:
Same as above.
IMO code that uses SigManager should first check for an invalid pid (coming from a corrupted message) and then proceed to use its methods. Therefore at this point invalid pids should be treated as errors.

Reply:
same